Tests/PDB/fragment_data/lib_10_z_5.txt

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
See discussion on https://github.com/biopython/biopython/pull/1191#issuecomment-305338212 and https://github.com/csblab/fragments - given this is now MIT licensed, we can probably include this small file, right @JoaoRodrigues ?

Reply:
Actually, please reuse ``Tests/PDB/lib_10_z_5.txt`` instead (already in the repository)